File: data/imported/knowledge/slides/java/merged/snippets/snippet_080.java
@Test
    void testClearShapes() {
        try (var pres = new Presentation()) {
            var slide = pres.getSlides().get(0);
            slide.getShapes().addAutoShape(ShapeType.RECTANGLE, 50, 50, 200, 100);
            slide.getShapes().clear();
            assertThat(slide.getShapes().size()).isEqualTo(0);
        }
    }

File: data/imported/knowledge/slides/net/merged/snippets/snippet_080.cs
[Fact]
    public void TestIterateShapes()
    {
        // Shapes collection is iterable.
        using var pres = new Presentation();
        var slide = BlankSlide(pres);
        slide.Shapes!.AddAutoShape(ShapeType.Rectangle, 50, 50, 200, 100);
        slide.Shapes!.AddAutoShape(ShapeType.Ellipse, 300, 50, 150, 150);
        var shapes = slide.Shapes!.ToList();
        shapes.Count.Should().Be(2);
    }